apps/src/sites/studio/pages/pd/regional_partner_contact/new.js

```diff
@@ -1,11 +0,0 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import RegionalPartnerContact from '@cdo/apps/code-studio/pd/regional_partner_contact/RegionalPartnerContact';
```

Comment:
Looks like this was the only place we imported `RegionalPartnerContact.jsx`, so you could remove that file now too.

Reply:
That makes sense -- there are a bunch of tests associated with that component, so I was going to remove in the follow-up (I was able to get this first PR together without messing with tests).